external/storm-autocreds/src/main/java/org/apache/storm/hbase/security/HBaseSecurityUtil.java

```diff
@@ -67,7 +67,7 @@ public static UserProvider login(Map conf, Configuration hbaseConfig) throws IOE
             }
             return legacyProvider;
         } else {
-            return UserProvider.instantiate(hbaseConfig);
+            return null;
```

Comment:
Wont this throw NPE in the caller ?

check: https://github.com/apache/storm/blob/1.x-branch/external/storm-hbase/src/main/java/org/apache/storm/hbase/trident/state/HBaseMapState.java#L84